ztp/source-crs/PerformanceProfile2.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: performance.openshift.io/v2
```

Comment:
The file should be named after its intended use. Having a "2" version doesn't indicate when this should be used vs the original. Same comment for other "2" versions below.

Reply:
I rename the files with -SetSelector , open for any  other name that make sense.